srtcore/core.cpp

```diff
@@ -3582,7 +3582,16 @@ SRTSOCKET CUDT::makeMePeerOf(SRTSOCKET peergroup, SRT_GROUP_TYPE gtp, uint32_t l
     }
     else
     {
-        gp = &newGroup(gtp);
+        try
+        {
+            gp = &newGroup(gtp);
+        }
+        catch (...)
```

Comment:
Waiting for any certain exception or any?

Reply:
Probably only system exceptions and those freshly added here (although "added" is only in order to comply with the C++ std version).